comfy_cli/command/run.py

```
@@ -146,30 +435,47 @@ def execute(
-            "[yellow]For long-running workflows, increase the timeout: comfy run --workflow <file> --timeout 300[/yellow]"
-        )
+        if json_mode:
+            emitter.emit_failed(
+                "timeout",
+                f"WebSocket timed out after {timeout}s waiting for server response",
+                timeout_seconds=float(timeout),
+            )
+        else:
+            pprint(
+                f"[bold red]Error: WebSocket timed out after {timeout}s waiting for server response.[/bold red]\n"
+                "[yellow]For long-running workflows, increase the timeout: comfy run --workflow <file> --timeout 300[/yellow]"
```

Pylint (4.0.5)

[convention] Line too long (126/120)

(C0301)